### DISCUSSION TITLE
27” Apple Cinema Display issues

### DISCUSSION THREAD

**Discussion by evanthorpe** [MonitorControl/MonitorControl#896] [Q&A]
Hi! I’ve got a 27” Apple Cinema Display (mini DisplayPort, not Thunderbolt), and I can’t seem to get it up to its max brightness. Looks about 80% of the way there to my eye. MC seems to be working to the point where I can now use keyboard buttons to adjust the brightness (didn’t work out if the box that way…I’m using a 14” M1 Pro with Monterey)…but now it does.

However, I can’t seem to get its brightness to the right level. I also have an Apple Thunderbolt Display and this looks about 20% dimmer. Putting a value of 255 in the DDC field doesn’t seem to do anything…in fact, nothing in that field makes a difference…meaning, if I set it to 50 it doesn’t do anything. I’ve followed the recommendation of disabling some settings in the General tab…still no difference :/ Any help?

**Comment by waydabber** [comment]
Does the brightness change at all f you disable "Combine hardware and software brightness?"

**Reply by evanthorpe** [reply]
No. I have it currently "unchecked" in the General tab and nothing seems to affect the brightness.

![image](https://user-images.githubusercontent.com/4305730/149678226-4bf3de7d-b6c6-4f35-8880-f91c31d75354.png)

![image](https://user-images.githubusercontent.com/4305730/149678231-a08745f8-dd08-4d37-bce2-6e5001c857dd.png)

![image](https://user-images.githubusercontent.com/4305730/149678237-285f6e30-ea7c-4b23-8d35-713568c3a52b.png)

**Reply by evanthorpe** [reply]
There's a good shot of my settings so far. I did have Lunar installed to try that, so I'm not sure if that's conflicting with it. I "reset all settings" within Lunar to try and start from scratch, but if there's something there blocking MC from working, I can disable it or change the settings there!

**Reply by evanthorpe** [reply]
Again, MC seems to have given the Cinema Display the ability to use the brightness keys on my computer to control it...but maxing out brightness there doesn't seem to be maxing out the monitor brightness. And I'm wondering if the lack of response in the DDC max field is indicating that as well.

I tried to get the m1ddc Github tool that you built to work...but I'm afraid it seems to be a bit over my head when it comes to running programs and using Terminal? Obviously, MC would be preferred since it's a bit simpler in the user interface. Thanks!

**Reply by the0neyouseek** [reply]
You seem to have DDC control disabled fo the monitor which would make changing DDC Max have no effect, could you try enabling "Use hardware DDC control" and see if it fix your issue

**Reply by evanthorpe** [reply]
Okay, just checked it...but still no change in brightness (brighter, dinner, etc) when changing values. Uploading a screen grab (which obviously wouldn't reflect brightness changes, but you can see what I'm doing) and nothing I'm doing in that video seems to have an affect on the screen brightness.

https://user-images.githubusercontent.com/4305730/149680264-fb9459bf-4d0e-48cf-bcda-4077f4ce555d.mov

**Reply by evanthorpe** [reply]
Here are my device/OS specifics: 

![image](https://user-images.githubusercontent.com/4305730/149680294-0d6b4667-cc6d-4fe8-9bc1-e08a08e45cbf.png)

**Reply by the0neyouseek** [reply]
Disabling "hardware DDC control" will make MonitorControl use "Software dimming" those not changing the display brightness but just the color ouput to dim the screen. If you are you using the HDMI port of the Macbook pro there's a known issue where Apple is blocking DDC (hardware) control. Could you try with an usb-c/thunderbolt cable ?

**Reply by evanthorpe** [reply]
So what I have is an Apple Cinema Display (which has the non-Thunderbolt mini-displayport cable), and that goes to a mini-Display port -> Displayport adapter into an Anker dock that has Displayport input. That runs to my Macbook Pro with a single USB C cable. So:

Display - mini-Displayport to Displayport adapter - Anker 13-in-1 dock ([More info link here](https://us.anker.com/products/a8392?currency=USD&variant=37438617944214&utm_source=google&utm_medium=shopping&utm_campaign=us_anker_conversion_alltime_shopping_hub_purchase_ost&utm_content=AnkerHub&utm_term=%7B12231402696%7D_%7B117620347656%7D_%7B495828514211%7D&gclid=Cj0KCQiAoY-PBhCNARIsABcz772fHW-UM87DOm3VZ8ifMLw1FkXaP8x7hr4RdOpIRFr3t60hqWjCaXUaApaWEALw_wcB)) - USB C cable - M1 Pro.

**Reply by evanthorpe** [reply]
I'm not using HDMI in any way or Thunderbolt in any way right now. Just the mini DP to DP adapter and dock. 

I don't have a straight "mini-DP to USB C" cable so that I could go straight into the M1 Pro...but that's less than ideal anyways since it'd be two cables running into my MacBook at that point (if that makes sense).
